<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>1081</width>
     <height>911</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>951</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2b3038;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #20242a;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #ff662b;
color: rgb(212, 192, 169);
paddinf-bottom: 7px;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>listening...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:  rgb(212, 192, 169);
color: #20242a;
border-radius: 10px;
}
QPushButton:hover{
background-color: #837c73;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>Start listening</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>941</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(212, 192, 169);</string>
    </property>
    <property name="text">
     <string>&lt;strong&gt;KASTA&lt;/strong&gt; VA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_description">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>951</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #837c73</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Virtual Assistant&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="endButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>540</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:  rgb(212, 192, 169);
color: #20242a;
border-radius: 10px;
}
QPushButton:hover{
background-color: #837c73;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>Stop listening</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #2b3038;
color: #20242a;
border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../icons/help_icon.png</normaloff>../../icons/help_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userEmailLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #837c73</string>
    </property>
    <property name="text">
     <string>xdxdxd</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #837c73</string>
    </property>
    <property name="text">
     <string>xdxdxd</string>
    </property>
   </widget>
   <widget class="QLabel" name="kastaLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>680</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #ff662b;
color: rgb(212, 192, 169);
paddinf-bottom: 7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/nowyPrzedrostek/kaastavector.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../icons/res.qrc">:/nowyPrzedrostek/kaastavector.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_description_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #837c73</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Helper&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addCommandButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:  rgb(212, 192, 169);
color: #20242a;
border-radius: 10px;
}
QPushButton:hover{
background-color: #837c73;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>Add command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myCommandsButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:  rgb(212, 192, 169);
color: #20242a;
border-radius: 10px;
}
QPushButton:hover{
background-color: #837c73;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>My commands</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeTextEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>341</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #ff662b;
color: rgb(212, 192, 169);
paddinf-bottom: 7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>enter command...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterCommandButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:  rgb(212, 192, 169);
color: #20242a;
border-radius: 10px;
}
QPushButton:hover{
background-color: #837c73;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>Enter command</string>
    </property>
   </widget>
   <widget class="QLabel" name="listeningLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #837c73</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>470</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #20242a;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="keyboardLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>370</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #2b3038;
color: #20242a;
border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../icons/keyboardicon.png</normaloff>../../icons/keyboardicon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="speakerIcon">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #2b3038;
color: #20242a;
border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../icons/speaker.png</normaloff>../../icons/speaker.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>emailLabel</zorder>
   <zorder>startButton</zorder>
   <zorder>label_title</zorder>
   <zorder>label_description</zorder>
   <zorder>endButton</zorder>
   <zorder>helpButton</zorder>
   <zorder>userEmailLabel</zorder>
   <zorder>userNameLabel</zorder>
   <zorder>kastaLabel</zorder>
   <zorder>label_3</zorder>
   <zorder>label_description_3</zorder>
   <zorder>addCommandButton</zorder>
   <zorder>listeningLabel</zorder>
   <zorder>label_4</zorder>
   <zorder>typeTextEdit</zorder>
   <zorder>myCommandsButton</zorder>
   <zorder>enterCommandButton</zorder>
   <zorder>keyboardLabel</zorder>
   <zorder>speakerIcon</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../icons/res.qrc"/>
 </resources>
 <connections/>
</ui>
